.github/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contribution Guidelines
+
+Thank you for your interest in contributing to C-Breez!
+
+We welcome contributions from everyone and appreciate your help in making C-Breez better.
+
+## How to Contribute
+
+### Communication First
+
+**Always communicate before coding!**
```

Comment:
Can you rephrase it to be a bit friendlier? 🙂

Reply:
Removed.